Assets/Scripts/GameManager.cs

```diff
@@ -109,4 +109,41 @@ public void OpenJournal()
         }
         switchControlSystem();
     }
+
+    // Новый метод для джампскейра, который можно вызвать из триггера
```

Comment:
Please make your comments in english for this project as it's the teams shared language